api/server/router/network/network_routes.go

```diff
@@ -79,8 +80,16 @@ func (n *networkRouter) postNetworkCreate(ctx context.Context, w http.ResponseWr
 		return err
 	}
 
-	if _, err := n.clusterProvider.GetNetwork(create.Name); err == nil {
-		return libnetwork.NetworkNameError(create.Name)
+	if nw, err := n.clusterProvider.GetNetwork(create.Name); err == nil {
```

Comment:
I can understand your opinion. But even for production, people may try to create a network named "n1", "n2"..
It would be great if we can find a solution.